### PR TITLE
Display multiple images in Milestone component

### LINKED ISSUE #51: Multiple images in frontend Milestone component as carousel

### PR DESCRIPTION
**Pull request by lkeegan**
- milestone component cycles through available images for that milestone
- transition via opacity every 5secs
- provide list of image urls in demo data instead of a single image
- remove padding on mobile devices to use more of the screen
- resolves #51

### CODE REVIEW

File: frontend/src/lib/components/Milestone.svelte [hunk 1]
```diff
@@ -11,12 +11,18 @@
...
     // build list of possible image urls at build time to be able to dynamically use them at run time:
-    const images: Record<string, string> = import.meta.glob('$lib/assets/*.jpg', {
+    const img_urls: Record<string, string> = import.meta.glob('$lib/assets/*.jpg', {
         eager: true,
         query: '?url',
         import: 'default'
     });
 
+    let currentImageIndex:number = 0;
+    const imageInterval = 5000;
```
Comment (MaHaWo) [inline frontend/src/lib/components/Milestone.svelte:21]: is 5s perhaps a little long as a delay between images? I'm not sure where  a good delay would be that makes people realize that there is content to look at here before they get impatient and start doing things....

**Review by MaHaWo — APPROVED** [review]
lgtm. one minor remark wrt delay between images

File: frontend/src/lib/components/Milestone.svelte (hunk 1 above)
Comment (lkeegan) [reply frontend/src/lib/components/Milestone.svelte:21]: Yes I'll make an issue for this - it's currently a bit long if you're just clicking an answer, but if you open the dropdown help text and try to read it then (at least for me) I found it a bit annoying when it kept updating - could also just cycle once then stop I guess.

Comment (lkeegan) [reply frontend/src/lib/components/Milestone.svelte:21]: https://github.com/ssciwr/mondey-frontend-prototype/issues/62

**Pull request merged**